server/mlabns/conf/tools.csv

```diff
@@ -1,5 +1,6 @@
 tool_id,slice_id,http_port,show_tool_extra
 ndt,iupui_ndt,7123,False
+ndt_ssl,iupui_ndt,7123,False
```

Comment:
fakewww hasn't been updated to support SSL, returning fqdn:7123 won't make sense in the context of ndt_ssl, at least not until fakewww is also updated.  It might be better to populate the http_port field with the expected SSL port of 3010.  Also see note below about creating about the http:// protocol in _create_tool_url()